hw2_function/fastq_filtrator.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[51]:
+
+
+#Counting GC content
+def GC_comp(seq):
+    GC = ((seq.upper().count('C') + seq.upper().count('G'))/len(seq.strip()))*100
```

Comment:
1. Пробелы
2. Это хорошо читается, но не очень оптимально по производительности. count внутри себя делает полный проход по строке, и мы вызываем его два раза, хотя GC можно посчитать за один проход. Но это мелочи)

```suggestion
    GC = ((seq.upper().count('C') + seq.upper().count('G')) / len(seq.strip())) * 100
```